### DISCUSSION TITLE
EcoDim.07 unsupported

### DISCUSSION THREAD

**Discussion by c0mplex1** [Koenkk/zigbee2mqtt#7282] [Devices]
After upgrading from Zigbee2mqtt version 1.18.2 to version 1.18.3 my EcoDim.07 Zigbee Pro LED Dimmer is suddenly marked as unsupported device.
How can this be fixed?

**Comment by c0mplex1** [comment]
I have restored Zigbee2mqtt to the previous version 1.18.2-1 and the Dimmer is supported again.
I am not a software engineer, so I don't know how to fix the problem under version 1.18.3-1
Please help.

**Reply by Koenkk** [reply]
Duplicate of https://github.com/Koenkk/zigbee-herdsman-converters/issues/2538, already added back in the dev branch (https://www.zigbee2mqtt.io/how_tos/how-to-switch-to-dev-branch.html)

**Reply by c0mplex1** [reply]
I'm very sorry for the duplicate post. Didn't check it.
I can't get boiled any soup of the link you replied. :-(
I use Zigbee2mqtt on HA. What means under Hass.io addon "Use the edge version."
Please guide me. Sorry for my noob question.

**Reply by c0mplex1** [reply]
I have updated Zigbee2mqtt to version 1.18.3-1 again, but this didn't help.
The Dimmer is now unsupported again.
I don't know what to do to fix this as I do not understand what is written here:
https://www.zigbee2mqtt.io/how_tos/how-to-switch-to-dev-branch.html
Please help.

**Reply by c0mplex1** [reply]
Will the dimmer be automatically supported again with the next Zigbee2mqtt update?
Yes? Then I will wait until the next update comes out.

**Reply by Koenkk** [reply]
Yes, will be supported in the 1 June release.  For the dev branch on HA, install the edge version of the addon.

**Reply by Pliind** [reply]
@c0mplex1 As a fellow noob, justa as disorientated by the GitHub layout and such and a bit to many advanced terms for ones best comfort, I thank you for mentioning the dev-branch (as it sovled my issue), to enable it simple swap out your docker adress from koenkk/zigbee2mqtt to koenkk/zigbee2mqtt:latest-dev
Now Ecodim works for me again, icons are still gone but function is there. Hope it helps!

**Reply by c0mplex1** [reply]
@Koenkk @Pliind Thank you for your reply.
I will wait for the 1 June release, because I am too afraid to lose the registration of the 50+ Zigbee devices I have in the network.

**Reply by dvandonkelaar** [reply]
If you use [this addon](https://github.com/zigbee2mqtt/hassio-zigbee2mqtt), you can do the following:
1. stop the 'Zigbee2mqtt' addon
2. install the 'Zigbee2mqtt Edge' addon
3. copy the configuration from the 'Zigbee2mqtt' addon to the 'Zigbee2mqtt Edge' addon
4. start the 'Zigbee2mqtt Edge' addon

Yesterday, I performed this procedure and all my devices (including the EcoDim.07 dimmers) were linked again. As far as I know, it uses the 'Zigbee2mqtt' database and data folder in Home Assistant.

**Reply by c0mplex1** [reply]
What worry me is this info:
```
Updating the Edge add-on

To update the edge version of the add-on, you will need to uninstall and re-install the add-on.

⚠️ Make sure to backup your config as the procedure will not save this for you.

Steps

    Backup config from: Supervisor → Dashboard → Zigbee2mqtt Edge → Configuration
    Uninstall: Supervisor → Dashboard → Zigbee2mqtt Edge → Uninstall
    Refresh repo: Supervisor → Add-on store → ⋮ → Reload
    Re-install: Supervisor → Add-on store → Zigbee2mqtt Edge → Install
    Restore config to: Supervisor → Dashboard → Zigbee2mqtt Edge → Configuration
```
Do this mean that every time when there is a new release of Zigbee2mqtt this procedure must be executed?

**Reply by dvandonkelaar** [reply]
The explanation says to uninstall and re-install the **edge** addon. So yes, for every new release of the 'Zigbee2mqtt Edge' addon, you need to uninstall and re-install.
Personally, I wouldn't update unless something is broken, I needed my EcoDim.07 dimmers back and now they are back. No need to update for now.
